content/plotly/concepts/express/terms/box/box.md

```diff
@@ -0,0 +1,59 @@
+---
+Title: '.box()'
+Description: 'Creates box plots'
+Subjects:
+  - 'Computer Science'
+  - 'Data Science'
+Tags:
+  - 'Methods'
+  - 'Libraries'
+  - 'Data'
+  - 'Graphs'
+  - 'Plotly'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+  - 'paths/data-science'
+  - 'paths/data-science-foundations'
+---
+
+A box plot is a visual representation of the distribution of numeric data through quartiles. It uses boxes and lines to depict the distributions of one or more groups of numeric data. The box limits indicate the range of the central 50% of the data, with a central line marking the median value. In Plotly, the **`.box()`** method is present in the `plotly.express` high-level interface, is used to make box plots with one or two variables.
+
+## Syntax
+
+```pseudo
+plotly.express.box(data_frame=None, x=None, y=None, color=None, labels=None, title=None, ...)
+```
+
+- `data_frame`: The dataset containing the variables to be plotted.
+- `y`: The data for the y-axis.
+- `x` (Optional): The data for the x-axis
+- `color` (Optional): A column name in the data frame that determines the color of the boxes.
+- `labels` (Optional): Key-value pairs with the original column names as the key and the custom column names as the value.
+- `title` (Optional): The title of the box plot.
```

Comment:
Put this into the proper sequence (`y` is before `x`) and also check this once, because `x` and `y` are option parameters i guess. Please recheck this

Reply:
Actually, you have to give one of the parameter, either `x` or `y`.
For example,
```pseudo
fig = px.box(data_frame=df, title="The Total Bill Spent On Meals") -> Error
fig = px.box(data_frame=df, y="total_bill", title="The Total Bill Spent On Meals") -> Ok
fig = px.box(data_frame=df, x="time", title="The Total Bill Spent On Meals") -> Ok
fig = px.box(data_frame=df, x="time", y="total_bill", title="The Total Bill Spent On Meals") -> Ok
```
If we give one of them, another one automatically becomes optional.